app/server/movies.go

```diff
@@ -64,6 +57,38 @@ func (s *Server) serveMovie(w http.ResponseWriter, req *http.Request) {
 	s.serveFile(w, req, m.GetFile())
 }
 
+func (s *Server) serveMovieFile(w http.ResponseWriter, req *http.Request) {
+	vars := mux.Vars(req)
+	id := vars["id"]
+	name := vars["name"]
+
+	movie, err := s.library.GetMovie(id)
+	if err != nil {
+		s.renderError(w, req, err)
+		return
+	}
+
+	var path string
+	for _, f := range []struct {
+		suffix, path string
+	}{
+		{suffix: ".nfo", path: movie.NfoPath()},
+		{suffix: "poster.jpg", path: movie.MovieThumbPath()},
+		{suffix: "fanart.jpg", path: movie.MovieFanartPath()},
+	} {
+		if strings.HasSuffix(name, f.suffix) {
+			path = f.path
```

Comment:
We could break here?
```suggestion
			path = f.path
			break
```

Reply:
You're right, we can even simplify and make this code more correct by matching the exact file name:
```go
var path string
for _, p := range []string{
    movie.NfoPath(),
    movie.MovieThumbPath(),
    movie.MovieFanartPath(),
} {
    if name == filepath.Base(p) {
        path = p
        break
    }
}
```